### PR TITLE
Add saptune apply solution to netweavere formula

### PR DESCRIPTION
**Pull request by MalloZup**
add saptune to netweaver

### CODE REVIEW

**Comment by MalloZup** [comment]
I think this PR is already good but I'm trying to setup netweaver deployement right now to test it

File: pillar.example [hunk 1]
```diff
@@ -37,6 +37,11 @@ netweaver:
   # Enable operations in ASCS and ERS to set HA environment correctly (HA cluster is not installed)
   ha_enabled: True
 
+  # saptune solution to apply to all nodes ( by default nothing is applied)
+  # you can also use this to a single node if need to differ. see hana2 
+  # Warning: only a unique solution can exist into a node.
+  saptune_solution: 'HANA'
```
Comment (arbulu89) [inline pillar.example]: I would put `NETWEAVER` here as example

**Review by arbulu89 — APPROVED** [review]
@MalloZup Looks good! Just the comment about the example.
If you test and works it's ready to go!

File: pillar.example (hunk 1 above)
Comment (MalloZup) [reply pillar.example]: done thx

**Comment by MalloZup** [comment]
Ok i have managed to deploy and I could test this works fine. merging

**Pull request merged**